### ISSUE TITLE
Extending the list of available Quick Actions 

### ISSUE THREAD

**Issue by SigmaBonder** [deckerst/aves#390]
**Is your feature request related to a problem? Please describe.**
It would be nice to have all the actions available as Quick Actions buttons. Right now, to access some features, like rating or tagging, one has to slide up on a photo to reveal the buttons in the Info section. It's not ideal, as if one were to tag all the photos from past week, from example, it would take a lot of swipes: up to go to info section to see the buttons, down to go back again to the photo, right to go to the next one...

**Describe the solution you'd like**
All the actions that are possible for a photo/video should be available as Quick Actions. This includes tagging, rating and changing metadata, date or location.

**Describe alternatives you've considered**
An alternative is current behavior, which I consider not ideal due to the reasons outlined above.

**Additional context**
Not limiting the number of actions available would be nice as well, but that probably qualifies as a separate feature.

**Comment by deckerst** [comment]
Agreed. I already did what you suggest for the `Collection` page selection quick actions, allowing all the actions, including the editing ones. I should do that for the `Viewer` page as well.

>  Not limiting the number of actions available would be nice as well, but that probably qualifies as a separate feature.

I don't limit them, but they visually do not appear if your screen cannot accommodate them. Is that what you're talking about?

**Comment by SigmaBonder** [comment]
> I don't limit them, but they visually do not appear if your screen cannot accommodate them. Is that what you're talking about?

Yes. I would change the behavior to the following:

- add the toggle to enable/disable multi-line quick actions;
- if the toggle is disabled and user's screen is too narrow to accommodate more than N icons, limit the quick actions list to N icons in order to not confuse the user — possibly with a helpful tip.

My photos have 4:3 aspect ratio, so there is enough space for two lines of quick action buttons (and a third one would fit above the photo, under the basic metadata, but that would be probably overkill — though if this is a relatively simple thing to add and maintain, it could be worth considering).

**Comment by deckerst** [comment]
Thanks for the suggestions. I'm unlikely to implement multiple lines of quick actions, but I'll think about alternatives (maybe horizontal scrolling), and maybe a tip to avoid confusion, as you mentioned.

If you need many quick actions, one thing you can do is tilt your screen in landscape ^^
